step.yml

```diff
@@ -1,12 +1,33 @@
 title: "Git Clone Repository"
-summary: Clone a repository to the specified path on the VM
+summary: Clones your source code from a remote repository into a clean build machine on Bitrise.
```

Comment:
The step checks out the defined repository state, optionally updates the repository submodules and exports the achieved git repository state properties.